app/controllers/admin/bookings_controller.rb

```diff
@@ -24,4 +25,7 @@ def booking_params
     params.require(:booking).permit(:start_date, :duration)
   end
 
+  def all_bookings
+    @bookings = Booking.all.sort
```

Comment:
Tu pourrais remplacer cette méthode par un [scope](https://www.rubyguides.com/2019/10/scopes-in-ruby-on-rails/) pour l'occasion :)